### PR TITLE
chore(demo-playwright): test for `text-area`

### PR DESCRIPTION
**Pull request by AtharvChandratre**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [x] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Addresses https://github.com/taiga-family/taiga-ui/issues/4912

## What is the new behaviour?
This PR adds an e2e test (with playwright) for `text-area`

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo-playwright/tests/core/text-area/text-area.spec.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import {tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Textarea`, () => {
+    [`m`, `l`].forEach(size => {
+        test(`size of ${size}`, async ({page}) => {
+            await tuiGoto(
+                page,
+                `components/textarea/API?tuiMode=null&tuiTextfieldSize=${size}`,
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/core/text-area/text-area.spec.ts:9]:
```suggestion
                `components/textarea/API?tuiTextfieldSize=${size}`,
```

File: projects/demo-playwright/tests/core/text-area/text-area.spec.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            );
+            const example = page.locator(`#demo-content`);
+
+            await expect(example).toHaveScreenshot(
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/core/text-area/text-area.spec.ts:13]:
```suggestion
            const {apiPageExample} = new TuiDocumentationPagePO(page);

            await expect(apiPageExample).toHaveScreenshot(
```

File: projects/demo-playwright/tests/core/text-area/text-area.spec.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                `textarea-tuiTextfieldSize-${size}.png`,
+            );
+        });
+    });
+
+    test(`line break text`, async ({page}) => {
+        await tuiGoto(page, `components/textarea/API`);
+
+        const example = page.locator(`#demo-content tui-textarea textarea`).nth(1);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/core/text-area/text-area.spec.ts:22]:
```ts
const {apiPageExample} = new TuiDocumentationPagePO(page);

const textarea = apiPageExample.getByRole('textbox').nth(1);
```

File: projects/demo-playwright/tests/core/text-area/text-area.spec.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    [`m`, `l`].forEach(size => {
+        test(`size of ${size}`, async ({page}) => {
+            await tuiGoto(
+                page,
+                `components/textarea/API?tuiMode=null&tuiTextfieldSize=${size}`,
+            );
+            const example = page.locator(`#demo-content`);
+
+            await expect(example).toHaveScreenshot(
+                `textarea-tuiTextfieldSize-${size}.png`,
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/core/text-area/text-area.spec.ts:14]: The generated screenshot is too wide

Screenshot 2023-11-20 at 13 33 10

File: projects/demo-playwright/tests/core/text-area/text-area.spec.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    });
+
+    test(`line break text`, async ({page}) => {
+        await tuiGoto(page, `components/textarea/API`);
+
+        const example = page.locator(`#demo-content tui-textarea textarea`).nth(1);
+
+        await example.pressSequentially(`1\n2\n3\n4`);
+
+        await expect(example).toHaveScreenshot(`textarea-line-break.png`);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/core/text-area/text-area.spec.ts:26]: Check locally generated screenshot.
Fix them and pin the new version of it to this discussion, please.

File: projects/demo-playwright/tests/core/text-area/text-area.spec.ts (hunk 1 above)
Comment (AtharvChandratre) [reply projects/demo-playwright/tests/core/text-area/text-area.spec.ts:9]: Addressed

File: projects/demo-playwright/tests/core/text-area/text-area.spec.ts (hunk 2 above)
Comment (AtharvChandratre) [reply projects/demo-playwright/tests/core/text-area/text-area.spec.ts:13]: Addressed

File: projects/demo-playwright/tests/core/text-area/text-area.spec.ts (hunk 3 above)
Comment (AtharvChandratre) [reply projects/demo-playwright/tests/core/text-area/text-area.spec.ts:22]: Addressed

File: projects/demo-playwright/tests/core/text-area/text-area.spec.ts (hunk 4 above)
Comment (AtharvChandratre) [reply projects/demo-playwright/tests/core/text-area/text-area.spec.ts:14]: Addressed

File: projects/demo-playwright/tests/core/text-area/text-area.spec.ts (hunk 5 above)
Comment (AtharvChandratre) [reply projects/demo-playwright/tests/core/text-area/text-area.spec.ts:26]: Changed Accordingly

File: projects/demo-playwright/tests/core/text-area/text-area.spec.ts
Comment (nsbarsukov) [inline projects/demo-playwright/tests/core/text-area/text-area.spec.ts:1]: `TextArea` (`text-area`) was the previous naming of this component.

Now it is `Textarea` (`textarea`) is new naming (like native [`<textarea>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea)).

Let's rename file `text-area.spec.ts` to `textarea.spec.ts`, please.

Comment (AtharvChandratre) [reply projects/demo-playwright/tests/core/text-area/text-area.spec.ts:1]: Ah makes sense, renamed

File: projects/demo-playwright/tests/core/text-area/text-area.spec.ts [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Textarea`, () => {
+    test.beforeEach(async ({page}) => {
+        await page.setViewportSize({width: 600, height: 600});
```
[the remark this answers is not in the document]
Comment (AtharvChandratre) [reply projects/demo-playwright/tests/core/text-area/text-area.spec.ts:6]: Changed

**Review by nsbarsukov — APPROVED** [review]
Thank you!

**Pull request merged**